src/ddsrt/include/dds/ddsrt/events.h

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright(c) 2006 to 2020 ADLINK Technology Limited and others
```

Comment:
I do understand how it came to read "2006 to 2020", but I think for this calendar year a plain "2020" would be a better choice for a new file.

Reply:
Maybe I was a little to quick in just copy/paste-ing :)
Will change the date range from 2006-2020 to just 2020.

Reply:
This is now implemented.